backend/Shared/HitMeApp.Shared.Infrastructure/Exceptions/DefaultExceptionMapperRegistry.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace HitMeApp.Shared.Infrastructure.Exceptions
+{
+    internal sealed class DefaultExceptionMapperRegistry : IExceptionMapperRegistry
+    {
+        private readonly ConcurrentDictionary<string, IExceptionToResponseMapper> _exceptionMapperRegistry =
+            new ConcurrentDictionary<string, IExceptionToResponseMapper>();
```

Comment:
Make it of type `<Regex, IExceptionToResponseMapper>`. No need to use static `Regex.IsMatch` and implicitly create hundreds of `Regex` objects each time some exception is thrown. We can instantiate them once and then use their instance `.IsMatch` method.

Reply:
https://stackoverflow.com/questions/414328/using-static-regex-ismatch-vs-creating-an-instance-of-regex

Feasible in our scale as we're gonna have at most N+1 exception mappers, where N = number of modules.